--- sslsplit-0.5.5/pxyconn.c~	2019-08-30 13:09:00.000000000 +0200
+++ sslsplit-0.5.5/pxyconn.c	2021-10-03 21:51:40.305960755 +0200
@@ -2188,7 +2188,7 @@
 			/* these can happen due to client cert auth,
 			 * only log error if debugging is activated */
 			log_dbg_printf("Error from %s bufferevent: "
-			               "%i:%s %lu:%i:%s:%i:%s:%i:%s\n",
+			               "%i:%s %lu:%i:%s:%i:%s:%s\n",
 			               (bev == ctx->src.bev) ? "src" : "dst",
 			               errno,
 			               errno ? strerror(errno) : "-",
@@ -2199,24 +2199,22 @@
 			               ERR_GET_LIB(sslerr),
 			               sslerr ?
 			               ERR_lib_error_string(sslerr) : "-",
-			               ERR_GET_FUNC(sslerr),
 			               sslerr ?
 			               ERR_func_error_string(sslerr) : "-");
 			while ((sslerr = bufferevent_get_openssl_error(bev))) {
 				log_dbg_printf("Additional SSL error: "
-				               "%lu:%i:%s:%i:%s:%i:%s\n",
+				               "%lu:%i:%s:%i:%s:%s\n",
 				               sslerr,
 				               ERR_GET_REASON(sslerr),
 				               ERR_reason_error_string(sslerr),
 				               ERR_GET_LIB(sslerr),
 				               ERR_lib_error_string(sslerr),
-				               ERR_GET_FUNC(sslerr),
 				               ERR_func_error_string(sslerr));
 			}
 		} else {
 			/* real errors */
 			log_err_printf("Error from %s bufferevent: "
-			               "%i:%s %lu:%i:%s:%i:%s:%i:%s\n",
+			               "%i:%s %lu:%i:%s:%i:%s:%s\n",
 			               (bev == ctx->src.bev) ? "src" : "dst",
 			               errno,
 			               errno ? strerror(errno) : "-",
@@ -2227,18 +2225,16 @@
 			               ERR_GET_LIB(sslerr),
 			               sslerr ?
 			               ERR_lib_error_string(sslerr) : "-",
-			               ERR_GET_FUNC(sslerr),
 			               sslerr ?
 			               ERR_func_error_string(sslerr) : "-");
 			while ((sslerr = bufferevent_get_openssl_error(bev))) {
 				log_err_printf("Additional SSL error: "
-				               "%lu:%i:%s:%i:%s:%i:%s\n",
+				               "%lu:%i:%s:%i:%s:%s\n",
 				               sslerr,
 				               ERR_GET_REASON(sslerr),
 				               ERR_reason_error_string(sslerr),
 				               ERR_GET_LIB(sslerr),
 				               ERR_lib_error_string(sslerr),
-				               ERR_GET_FUNC(sslerr),
 				               ERR_func_error_string(sslerr));
 			}
 		}
